language/German/strings.po

```diff
@@ -1664,179 +1664,179 @@ msgstr "Menü zur Softwareauswahl überspringen"
 
 #: src/frontend/mame/ui/submenu.cpp:34
 msgid "Info auto audit"
-msgstr ""
+msgstr "Informationen über automatisches Audit"
 
 #: src/frontend/mame/ui/submenu.cpp:35
 msgid "Hide romless machine from available list"
-msgstr ""
+msgstr "Maschine ohne ROMs"
```

Comment:
Shouldn't this be 
"Machine ohne ROMs aus Liste der verfügbaren Maschinen ausblenden" ?
At least it is missing some information in the translation.